src/domain/usecases/GetAllSurveysUseCase.ts

```diff
@@ -41,7 +47,10 @@ export class GetAllSurveysUseCase {
                             .compact()
                             .value();
 
-                        return Future.success(filteredSurveys);
+                        return Future.success({
```

Comment:
@davidbgomes see the code above, the survey list is getting filtered. The page details need to reflect this filtering.